.ci/install_qemu.sh

```diff
@@ -48,17 +53,19 @@ uncompress_static_qemu() {
 }
 
 build_and_install_static_qemu(){
+	install_directory="/tmp"
```

Comment:
I think we can have this as a global variable as `/tmp` is used in other functions